exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -40,6 +40,29 @@
    */
   HashCode submitTransaction(TransactionMessage tx);
 
+  /**
+   * Returns a number of unconfirmed transactions those are currently located in
+   * the memory pool and are waiting for acceptance to a block.
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  int getUnconfirmedTransactions();
+
+  /**
+   * Returns <b>true</b> if the node is connected to the other peers.
+   * And <b>false</b> otherwise.
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  boolean healthCheck();
```

Comment:
more proper name?

Reply:
isNodeConnected? isNodeInNetwork?

BTW, is it OK to get an exception if the _client_ is not connected? (probably, yes, as we need to communicate different problems differently; but the `helthCheck` is kind of an innocent name)